opentelemetry-api/src/opentelemetry/propagator/__init__.py

```diff
@@ -0,0 +1,77 @@
+import typing
+
+import opentelemetry.context.propagation.httptextformat as httptextformat
+import opentelemetry.trace as trace
+from opentelemetry.context.propagation.tracestatehttptextformat import (
+    TraceStateHTTPTextFormat,
+)
+
+
+class Propagator:
```

Comment:
Does this need to be a class? Could't we make these free helper functions and have the selected formatters directly be globals? Or if we want to give API implementations more freedom, we should move the concept of formatters outside the SDK layer.

Reply:
helper functions would work fine here... there's nothing that an SDK has to override really, and this was more important when Context was an object that was passed into the constructor.

could do work to set httpformatters as global. I think ultimately the shape of globals will change anyway, don't particularly mind how configuration of httptextformatters look for this specific PR.